tests/util/yaml/test_init.py

```diff
@@ -53,6 +53,26 @@ def try_both_dumpers(request):
     importlib.reload(yaml_loader)
 
 
+@pytest.mark.parametrize(
+    ("yaml_content", "expected_output"),
+    [
+        ("", {}),
+        ("~", {}),
```

Comment:
Hmm this one is odd? I would expect it to be a NoneType value

Reply:
Yes, it's odd indeed. I added this test to show the behavior.